force-app/main/default/lwc/usefullLinks/usefullLinks.js

```diff
@@ -0,0 +1,11 @@
+import { LightningElement } from 'lwc';
+import { NavigationMixin } from 'lightning/navigation';
+
+export default class UsefullLinks extends NavigationMixin(LightningElement) {
+
+    fullLinks = [ 
```

Comment:
why is this called full links? what does that mean in this context a beetter name would be just links. because that is what that really is